ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+from gen_bufr2ioda_json import gen_bufr_json
+from wxflow import Logger, cast_strdict_as_dtypedict
+from wxflow import add_to_datetime, to_timedelta
+
+# Initialize root logger
+logger = Logger('run_bufr2ioda.py', level='INFO', colored_log=True)
+
+
+def bufr2ioda(CDATE, RUN, DMPDIR, config_template_dir, COM_OBS):
+    logger.info(f"Process {CDATE} {RUN} from {DMPDIR} to {COM_OBS} using {config_template_dir}")
+
+    # Derived parameters
+    PDY = CDATE[:8]
+    cyc = CDATE[8:10]
+
+    config = cast_strdict_as_dtypedict(os.environ)
```

Comment:
What is the point of passing arguments to this method, if os.environ is going to be invoked here.

Reply:
No point apart from my limited python skills leading me to copy what I see in other scripts.  Remove `os.environ`.  Done at [`2510b32`](https://github.com/NOAA-EMC/GDASApp/commit/2510b32a33a7b5060a227a83cebe002e797acdd6).